src/inject/dynamic-theme/style-manager.ts

```diff
@@ -162,7 +166,9 @@ export function manageStyle(element: StyleElement, {update, loadingStart, loadin
                 isStillLoadingError(accessError)
             ) {
                 try {
-                    await linkLoading(element);
+                    loadingLinkID = ++loadingLinkStyle;
```

Comment:
Hmm, looks like you have to use `const` here, so each style has its own ID. Also there is a naming convention to use `Id` instead of `ID` in JS and other similar languages, e.g. `document.getElementById`.